CMakeLists.txt

```diff
@@ -349,6 +348,7 @@ endif()
 if(CMAKE_PROJECT_NAME STREQUAL HERMES AND BUILD_TESTING)
   find_package(Catch2 REQUIRED)
   add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/test)
+  add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/adapter/test)
```

Comment:
Could you guard the building and testing of the adapters with a `HERMES_ENABLE_ADAPTERS` CMake option? The default can be `ON`, but it would be nice to disable the adapters when doing library development.

Reply:
added option HERMES_ENABLE_ADAPTERS.